src/include/migraphx/config.hpp

```diff
@@ -24,10 +24,17 @@
 #ifndef MIGRAPHX_GUARD_CONFIG_HPP
 #define MIGRAPHX_GUARD_CONFIG_HPP
 
+#include <migraphx/version.h>
 namespace migraphx {
 
 #if !defined(MIGRAPHX_USE_CLANG_TIDY) && !defined(DOXYGEN)
-#define MIGRAPHX_INLINE_NS version_1
+
+#define MIGRAPHX_CONCAT_1(x, y) x##y
+#define MIGRAPHX_CONCAT(x, y) MIGRAPHX_CONCAT_1(x, y)
+
+#define MIGRAPHX_VERSION MIGRAPHX_CONCAT(MIGRAPHX_VERSION_MAJOR, MIGRAPHX_VERSION_MINOR)
```

Comment:
This should include the patch and tweak version.

Reply:
Ah wait that needs to be added to the version.h header. This should probably be done in a seperate PR.